<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>570</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>313</y>
      <width>34</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0);
color: rgb(186, 186, 186);
border: none;</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>71</y>
      <width>171</width>
      <height>136</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(103, 151, 72);
color: rgb(0, 0, 0);
border: none;
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>276</y>
      <width>65</width>
      <height>53</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(186, 186, 186);
border: none;</string>
    </property>
    <property name="text">
     <string>B</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>364</y>
      <width>35</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0);
color: rgb(186, 186, 186);
border: none;</string>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftButton">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>338</y>
      <width>30</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0);
color: rgb(186, 186, 186);
border: none;</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightButton">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>338</y>
      <width>32</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0);
color: rgb(186, 186, 186);
border: none;</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aButton">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>252</y>
      <width>65</width>
      <height>53</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(186, 186, 186);
border: none;
</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>448</y>
      <width>32</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 119, 119);
color: rgb(186, 186, 186);
border: none;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>447</y>
      <width>33</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 119, 119);
color: rgb(186, 186, 186);
border: none;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="Background">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>71</y>
      <width>82</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(103, 151, 72);
color: rgb(0, 0, 0);
border: none;
</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>106</y>
      <width>82</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(103, 151, 72);
color: rgb(0, 0, 0);
border: none;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>82</width>
      <height>67</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(103, 151, 72);
color: rgb(0, 0, 0);
border: none;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>349</y>
      <width>65</width>
      <height>53</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(186, 186, 186);
border: none;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yButton">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>325</y>
      <width>65</width>
      <height>53</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(186, 186, 186);
border: none;</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <zorder>Background</zorder>
   <zorder>upButton</zorder>
   <zorder>textEdit</zorder>
   <zorder>downButton</zorder>
   <zorder>leftButton</zorder>
   <zorder>rightButton</zorder>
   <zorder>aButton</zorder>
   <zorder>selectButton</zorder>
   <zorder>startButton</zorder>
   <zorder>bButton</zorder>
   <zorder>textEdit_2</zorder>
   <zorder>textEdit_3</zorder>
   <zorder>listWidget</zorder>
   <zorder>xButton</zorder>
   <zorder>yButton</zorder>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
